packages/nightingale-linegraph-track/src/nightingale-linegraph-track.js

```diff
@@ -192,14 +192,23 @@ class NightingaleLinegraphTrack extends ProtvistaTrack {
   drawLine(d) {
     const curve = d.lineCurve || "curveLinear";
 
-    return line()
+    var graph;
```

Comment:
Please use `let`|`const`
```suggestion
    let graph;
```